### PR TITLE
Documention demo apps

### LINKED ISSUE #12: [Handover] Detail the Demo apps

### PR DESCRIPTION
**Pull request by Vicba**
Fixes #12 

- [x] MockBook
- [x] DoctorApp

### CODE REVIEW

File: toco/doctorapp/index.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# DoctorApp
+
+MockBook is a social network that allows users to view their posts.
+The goal is to showcase how changing the permissions in Loama can affect the app in accessing the resources.
```
Comment (Zelzahn) [inline toco/doctorapp/index.md]:
```suggestion
The goal is to showcase how changing the permissions in Loama can affect how resources are accessed by the app.
```

File: toco/doctorapp/index.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Profile information: name, email, bio, profile picture
+- posts: text, image, video
+
+All data is retrieved from the user's pod using the functions provided by the [Loama controller](/deliverables/controller.md).
+You can find the current limitations of the app in the [limitations](#current-limitations) section.
+
+## How to run?
+
+Before running the application, make sure you have the code of Loama in the same folder as you have for this repository.
```
Comment (Zelzahn) [inline toco/doctorapp/index.md]:
```suggestion
Before running the application, make sure you have the code of Loama in the same directory as this repository.
```

File: toco/doctorapp/index.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# DoctorApp
+
+MockBook is a social network that allows users to view their posts.
```
Comment (Zelzahn) [inline toco/doctorapp/index.md]: Copypasta tsk tsk tsk

Comment (elroyK) [inline toco/doctorapp/index.md]: title says DoctorApp, breadcrumbs say Doctorapp, page says Mockbook?

File: toco/doctorapp/index.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+The goal is to showcase how changing the permissions in Loama can affect the app in accessing the resources.
+
+It currently displays the following information:
+
+- Profile information: name, email, bio, profile picture
+- posts: text, image, video
```
Comment (elroyK) [inline toco/doctorapp/index.md]: either capitalize both or none of them for consistency :)

File: toco/mockbook/index.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
+# Mockbook
+
+MockBook is a social network that allows users to view their posts.
+The goal is to showcase how changing the permissions in Loama can affect the app in accessing the resources.
+
+It currently displays the following information:
+
+- Profile information: name, email, bio, profile picture
+- posts: text, image, video
```
Comment (elroyK) [inline toco/mockbook/index.md]: see comment on the previous file :)

**Pull request merged**